go.mod

```diff
@@ -7,6 +7,9 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-bindata/go-bindata/v3 v3.1.3
 	github.com/google/cel-go v0.12.4
+	github.com/h2non/filetype v1.1.3
+	github.com/h2non/go-is-svg v0.0.0-20160927212452-35e8c4b0612c
+	github.com/joelanford/ignore v0.0.0-20210610194209-63d4919d8fb2
```

Comment:
We should probably bring the ignore package in here instead of importing it from Joe's, doesn't look like it'll be too much work: https://github.com/joelanford/ignore/blob/main/ignore.go

Reply:
Would be great to create an issue before we merge this PR so that we don't loose track of this

Reply:
The reason this is separate is because `openshift/oc` needs it too, and `oc` does not import `operator-framework` stuff (or at least isn't currently).

If we move that ignore package here, then we need to do one of the following:
1. Maintain the code separately for `oc` somewhere.
2. Get `oc` to agree to require this API repo.

Thread about it here: https://github.com/openshift/oc/pull/868/files#r668149063